app/.env.local.example

```diff
@@ -1,45 +0,0 @@
-# Rename file to .env.local (or .env) and populate values
```

Comment:
Why was this file removed?

Reply:
Me being a n00b. I just renamed it, and didn't copy it.